build.sbt

```diff
@@ -781,7 +782,7 @@ lazy val Dependencies = new {
   }
 
   object Decline {
-    val core = Def.setting("com.monovore" %%% "decline" % "2.3.0")
+    val core = Def.setting("com.monovore" %%% "decline" % "2.3.1")
     val effect = Def.setting("com.monovore" %%% "decline-effect" % "2.3.0")
```

Comment:
should we bump the one below also? or introduce a val to share

Reply:
good catch, it is indeed used in docs. I'll bring in a val